defs.h

```diff
@@ -91,8 +91,14 @@
 #define PAL16_RED		224
 #define PAL16_GRAY		240
 
-#define SCREEN_WIDTH	640
-#define SCREEN_HEIGHT	480
+#define SCREEN_WIDTH	1280

```

Comment:
I've read somewhere that width should be a multiple of 64 and Height of 32.
May be you can add a comment here or define something like
SCREEN_WIDTH  (640 & 0xFFC0)
SCREEN_HEIGHT (480 & 0xFFE0 )

Reply:
`	mx = ViewX + tx + ty - (zoomflag ? SCREEN_WIDTH / 64 : SCREEN_WIDTH / 128);` - just found out why width has to be divisible by 64 :P

Reply:
The rendering goes horribly wrong if it's not dividable by 32

Reply:
@AJenbo SDL adds "pitch" to textures -- padding pixels in every row so that rows align with 4-byte boundaries (on x84). The rendering code likely isn't taking that into account.

Reply:
@glebm looks like you might be right, at least it works with 1144x480 which is dividable by 8 but not 32 or 64. Not sure if it's worth trying to deal with atm or if we should just wait till it's rewritten using SDL.